src/Enums.js

```diff
@@ -10,8 +10,8 @@ export const memberApplicationState = {
 }
 
 export const membershipPlans = {
-  YEAR: 1,
-  SEMESTER: 2
+  YEAR: 2,
```

Comment:
Why were these switched?

Reply:
YEAR: should register 2 semester
SEMESTER: should register 1 semester

Since the api/register look at these number to calculate an expiration date. So 1 and 2 are the number of semesters of that option